scaffolding-scaffolding/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_name=scaffolding-scaffolding
```

Comment:
How about `scaffolding-base`, `scaffolding-common`, or `scaffolding-core`? Those tend to be the words we use when we describe a shared library in the Habitat codebase